site/docs/latest/admin/http.md

```diff
@@ -0,0 +1,317 @@
+---
+title: BookKeeper http endpoints
+---
+
+This document is to introducing BookKeeper http endpoints that could be used for BookKeeper administration.
+If user want to use this feature, parameter "httpServerEnabled" need to be set to "true" in file conf/bk_server.conf.
+
+## All the endpoints
+
+Currently all the http endpoints could be divided into these 4 components:
```

Comment:
The HTTP endpoints are divided into the following groups:

Reply:
Perhaps they should be grouped by endpoints which are bookie specific and those which are for the whole cluster.

Reply:
Thanks. will change the words. 
And it is a good idea to do a new grouping following your suggestion. will consider to do it, and opened a new issue #722 to tracking this, but as the requirements goes on, maybe 2 big groups need futher dividing.